### PR TITLE
LRDOCS-8420 supported

### PR DESCRIPTION
**Pull request by dantewang**

### CODE REVIEW

File: docs/dxp/7.x/en/developing-applications/reference/portlet-descriptor-to-osgi-service-property-map.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ The standard portlet descriptor mappings are first.
 |`/liferay-portlet-app/portlet/header-request-attribute-prefix`|`com.liferay.portlet.header-request-attribute-prefix=<String>` [7](#seven)|
 |`/liferay-portlet-app/portlet/header-timeout`|`header-timeout=<int>`|
 |`/liferay-portlet-app/portlet/icon`|`com.liferay.portlet.icon=<String>`|
-|`/liferay-portlet-app/portlet/include`|not supported|
+|`/liferay-portlet-app/portlet/include`|`com.liferay.portlet.include=<boolean>`|
```
Comment (dantewang) [inline docs/dxp/7.x/en/developing-applications/reference/portlet-descriptor-to-osgi-service-property-map.md:110]: See https://github.com/liferay/liferay-portal/blob/bc74ba288a5f79488f5a72890162b742cee9d8bc/modules/apps/static/portal-osgi-web/portal-osgi-web-portlet-tracker/src/main/java/com/liferay/portal/osgi/web/portlet/tracker/internal/PortletTracker.java#L696

This property affects the value of `com.liferay.portal.kernel.model.Portlet.isInclude()` which is used in many places.

Comment (dantewang) [inline docs/dxp/7.x/en/developing-applications/reference/portlet-descriptor-to-osgi-service-property-map.md:110]: Read by https://github.com/liferay/liferay-portal/blob/bc74ba288a5f79488f5a72890162b742cee9d8bc/modules/apps/static/portal-osgi-web/portal-osgi-web-portlet-tracker/src/main/java/com/liferay/portal/osgi/web/portlet/tracker/internal/PortletTracker.java#L696

File: docs/dxp/7.x/en/developing-applications/reference/portlet-descriptor-to-osgi-service-property-map.md [hunk 2]
```diff
@@ -81,6 +81,7 @@ The standard portlet descriptor mappings are first.
 |`/liferay-portlet-app/portlet/active`|`com.liferay.portlet.active=<boolean>`|
 |`/liferay-portlet-app/portlet/add-default-resource`|`com.liferay.portlet.add-default-resource=<boolean>`|
 |`/liferay-portlet-app/portlet/ajaxable`|`com.liferay.portlet.ajaxable=<boolean>`|
+|`/liferay-portlet-app/portlet/application-type`|`com.liferay.portlet.application-type=full-page-application` and/or `com.liferay.portlet.application-type=widget`|
```
Comment (dantewang) [inline docs/dxp/7.x/en/developing-applications/reference/portlet-descriptor-to-osgi-service-property-map.md:84]: @jhinkey For actual example of the usage see https://github.com/liferay/liferay-portal/blob/b75cf72ed573622f5c2fff167a5c8b58904414ca/modules/apps/blogs/blogs-web/src/main/java/com/liferay/blogs/web/internal/portlet/BlogsPortlet.java#L41, this option only supports 2 values `full-page-application` and `widget` but one can declare both as shown in `BlogsPortlet`.

**Comment by dantewang** [comment]
@jhinkey Changes in this pull is based on this version of [PortletTracker](https://github.com/liferay/liferay-portal/blob/bc74ba288a5f79488f5a72890162b742cee9d8bc/modules/apps/static/portal-osgi-web/portal-osgi-web-portlet-tracker/src/main/java/com/liferay/portal/osgi/web/portlet/tracker/internal/PortletTracker.java). Although it's from master, there's no property related change in at least one year from its git history. So I think it's safe to base on it.

**Comment by dantewang** [comment]
@jhinkey I will push new commits to this pull when I find more mismatching items. I'll add another comment to let you know when I finish.

**Comment by jhinkey** [comment]
Thanks @dantewang ! I'll wait for further comments.

**Comment by dantewang** [comment]
@jhinkey Done. It's based on `PortletTracker` and other places which also reads these properties from `Portlet` OSGi services.

**Comment by jhinkey** [comment]
That's great, @dantewang ! Thanks

**Comment by jhinkey** [comment]
Started reviewing. Thanks.

**Comment by jhinkey** [comment]
Sent https://github.com/sez11a/liferay-learn/pull/252

**Pull request closed without merging**